utils/src/check/super.c

```diff
@@ -262,8 +262,19 @@ int check_supers(int data_fd, bool repair)
 			SCOUTFS_FORMAT_VERSION_MAX);
 
 	debug("Quorum Config Version: %llu", global_super->qconf.version);
-	if (global_super->qconf.version != 1)
+	if (global_super->qconf.version != 1) {
 		problem(PB_QCONF_WRONG_VERSION, "Wrong Version: %llu (expected 1)", global_super->qconf.version);
+		if(repair) {
```

Comment:
space

Reply:
There's still a missing space here unless github is showing me something outdated...

Reply:
Yea I think that might be it, hopefully my next push will fix it

Reply:
Okay I think I fixed it, it seems I fixed it in a later branch but never applied it here, should be fixed her in a second